src/utility.js

```diff
@@ -53,14 +53,15 @@ function parseUncssrc(filename) {
 function parsePaths(source, stylesheets, options) {
     return stylesheets.map(function (sheet) {
         var sourceProtocol;
+        var isLocalFile = sheet.substr(0, 5) === 'file:';
```

Comment:
Should we be using url.parse(sheet).protocol (to check for both file and http) here?